### ISSUE TITLE
download the dataset

### ISSUE THREAD

**Issue by tongpinmo** [tinghuiz/SfMLearner#68]
the dataset is too big which take a lot of time

**Comment by tinghuiz** [comment]
Sorry we like it big :)

**Comment by tongpinmo** [comment]
ok,actually ,i have run the project,works well,but i have a question,why the result of some supervised method on depth or pose didn't work well(Table 1 Eigen et),and another problem,your predicting on Figure 5,6,8,we can just see the predicted outline which can't apply to the actual situation
more,the predicted depth method ,if we make use of the relationship of the pixel frame,maybe a modest performance?

**Comment by tinghuiz** [comment]
I think there might be two reasons: 1) architecture difference and 2) the ground-truth depth points are sparse (i.e. not enough training data). The paper below showed that with extra depth supervision it's possible to get better results: https://arxiv.org/abs/1702.02706. 

Sorry I don't understand the second part of your question. Could you rephrase?

**Comment by tongpinmo** [comment]
ok ,thank you very much ,the second question ,i mean we can just see the outline of objects,like cars,trees,but can it be applied to actual situation?for example,automatic driving
